db/c.cc

```diff
@@ -5072,6 +5072,16 @@ rocksdb_fifo_compaction_options_t* rocksdb_fifo_compaction_options_create() {
   return result;
 }
 
+void rocksdb_fifo_compaction_options_set_allow_compaction(
+    rocksdb_fifo_compaction_options_t* fifo_opts, bool allow_compaction) {
```

Comment:
Although `bool` works as far as I can tell, almost everywhere else uses `unsigned char` in this file and I found it's documented in the header - https://github.com/facebook/rocksdb/blob/24ac53d81a2c97f6c30305bd2c3a12f0800ab945/include/rocksdb/c.h#L40. Do you mind changing to that for consistency?

Reply:
Indeed! Thank you!